src/aiovodafone/api.py

```diff
@@ -353,6 +379,8 @@ async def get_docis_data(self) -> dict[Any, Any]:
         """Get docis data."""
         _LOGGER.debug("Get docis data")
         response = await self._get_page_result("/api/v1/sta_docsis_status")
+        if not response:
+            raise RequestFailed("Failed to get DOCSIS status")
```

Comment:
`docis` or `docsis` ? Small letter or capital letters ? 

Please recheck typo and be consistent

Reply:
Reworked